modules/core/shared/src/main/scala/lucuma/core/math/dimensional/unit.scala

```diff
@@ -38,7 +49,7 @@ trait Units { self =>
     case _           => false
   }
 
-  override def hashCode: Int = Objects.hash(name, abbv)
+  override def hashCode: Int = Objects.hash(name, abbv, serialized)
```

Comment:
Do you need to ask override `equals`?

Reply:
You are right, `serialized` should take part in both `hashCode` and `equals` or none.